.env.testing

```diff
@@ -0,0 +1,39 @@
+APP_NAME=Laravel
+APP_ENV=local
+APP_KEY=base64:hRdl2Nk5p7X0IUhPlBJPmsm0xhJP+pQMai22zt/uroA=
+APP_DEBUG=true
+APP_URL=http://localhost
+LOG_CHANNEL=stack
+
+DB_CONNECTION=sqlite
+DB_DATABASE=/home/jrbour/Desktop/school/mentalist/database/database_testing.sqlite
```

Comment:
Dont use local path.

Reply:
I will add this file in the gitignore file

Reply:
No it's ok, i fixed this